### PR TITLE
[ready-for-core-team-review]CRM-16189, Added Trial Balance report

### PR DESCRIPTION
**Pull request by pradpnayak**
---
- CRM-16189: Improve support for Accrual Method bookkeeping
  https://issues.civicrm.org/jira/browse/CRM-16189

### CODE REVIEW

File: CRM/Report/Form/Contribute/TiralBalance.php [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+ */
+
+/**
+ *
+ * @package CRM
+ * @copyright CiviCRM LLC (c) 2004-2016
+ * $Id$
+ *
+ */
+class CRM_Report_Form_Contribute_TiralBalance extends CRM_Report_Form {
```
Comment (yashodha) [inline CRM/Report/Form/Contribute/TiralBalance.php]: seems like a typo in the report name.

Comment (JoeMurray) [inline CRM/Report/Form/Contribute/TiralBalance.php]: @pradpnayak or @Edzelopez, please update to ..._TrialBalance

**Comment by JoeMurray** [comment]
@Edzelopez @pradpnayak please address Jenkins' errors

**Comment by Edzelopez** [comment]
I have addresses Jenkins errors for the database fatal error. The current error is due to incomplete report not generating sql for rows. The label is WIP, so I'm assuming @pradpnayak has still to complete this.

**Comment by seamuslee001** [comment]
Looks like there is a syntax error when LIMIT is added to the Query

**Comment by seamuslee001** [comment]
Actually i'm wrong its lacking any sql is the issue also it appears it needs a rebasing

**Comment by JoeMurray** [comment]
Thanks @seamuslee001 for testing. I think it is marked WIP since more code needs to be added to make it functionally complete.

**Comment by JoeMurray** [comment]
@pradpnayak could you resolve conflicts and retest?

**Comment by monishdeb** [comment]
As per discussion with @pradpnayak am labelling this PR with [WIP]

**Comment by eileenmcnaughton** [comment]
I think this should be an extension @JoeMurray - this sort of fairly obscure report that most devs would not be qualified to debug seems to be an obvious fit for an extension. I expect more financial reports will come

As an aside - the file templates/CRM/Report/Form/Contribute/TrialBalance.tpl is not required if it is the default

**Comment by pradpnayak** [comment]
Closing this PR as the whole functionality will be moved to an extension

**Pull request closed without merging**